tests/wpt/web-platform-tests/fixme/detached-layout.html

```diff
@@ -0,0 +1,23 @@
+<!doctype html>
```

Comment:
This test can be moved to tests/wpt/mozilla/tests/mozilla. These are WPT tests that we don't upstream and are used for checking regressions in specific cases in our engine, rather than conformance tests for all engines.